recipes/ml_tooling/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "ml_tooling" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f1f370bbd6eae0d557e9ab28d8dd04c27b67c2d27e41dd08172a35e588e11ed1
+build:
+  number: 0
+  skip: true  # [py>=36]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - pandas
+    - numpy
+    - scikit-learn
+    - gitpython
+    - matplotlib
+    - pyyaml
+
+test:
+  imports:
+    - ml_tooling
+    - ml_tooling.plots
+
+about:
+  home: http://github.com/andersbogsnes/ml_tooling
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
Your license is not being packaged on ``pypi``. However, in tarballs, it is being packaged.
So, the easiest modification is to modify the source url to
```
https://github.com/andersbogsnes/{{ name }}/archive/{{ version }}.tar.gz
```

Reply:
Ah, so that's what's going wrong. I'll quickly try to update the pypi license_file metadata to see if that fixes it, else I'll change it to to your suggestion. Thank!

Reply:
I've updated the source on pypi, so I'm no longer getting the license error - not sure what these new failures are? They seem unrelated to my code, but I'm not sure :-)